csrc/predicate_compute.h

```diff
@@ -15,7 +15,29 @@
 #include <logical_domain_map.h>
 
 namespace nvfuser {
+// Struct holds the info returned from OneDimTmaLoadExpectArrive()
+struct OneDimTmaPredicateInfo {
```

Comment:
nit: generally public variable members should not use `_`. https://google.github.io/styleguide/cppguide.html#Variable_Names